src/styles/parts/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
```

Comment:
You should avoid using fixed values for margins. Consider using relative units like `em` or `rem` instead of `px`. This will make your layout more responsive.